Document GolangCI Support (#598)
cirruslabs/cirrus-ci-docs: docs/examples.md

  test_script: go test ./...
```

## GolangCI Lint

We highly recommend to configure some sort of linting for your Go project. One of the options is [Golang Lint](https://github.com/golangci/golangci-lint)

> which report format is supported by Cirrus CI annotator for providing inline reports on PRs. Here is an example of *GolangCI Lint*

Reviewer: ```suggestion
- the Cirrus CI annotator supports providing inline reports on PRs. Here is an example of a *GolangCI Lint*
```
